webrtc.html

```diff
@@ -1364,6 +1364,16 @@
             this value is <code>null</code>.
           </dd>
 
+          <dt>static sequence&lt;RTCIceServer&gt; getDefaultIceServers()</dt>
+
+          <dd>
+            <p>This method returns a list of ICE servers that are configured into
+            the browser.  A browser might be configured to use local or private
+            STUN or TURN servers.  This method allows an application to learn about
+            these servers and optionally use them.
+            </p>
+          </dd>
```

Comment:
I think this would make more sense on IceTransport.  It will still be valuable in WebRTC NV and it's ICE-specific.